front/src/applications/stdcm/utils.ts

```diff
@@ -61,7 +61,7 @@ export function getOperationalPointsWithTimes(simulationReport: SimulationReport
   }
 
   const { positions, times } = simulation.final_output;
-  const departureTime = extractTime(simulationReport.stdcmData.departure_time);
+  const departureTime = extractTime(dateTimeZToIso(simulationReport.stdcmData.departure_time));
```

Comment:
As the `departure_time` you receive from the endpoint is already in ISO8601 format (with the timezone), you don't need to convert it one more time. 
As you know that `departure_time` will be defined and in ISO format, to get the locale time of this date you can use the regular javascript Date api (and you can remove the new function you created in date.ts and the modifications you made to `extractTime`)

```suggestion
  const departureTime = new Date(simulationReport.stdcmData.departure_time).toLocaleTimeString();
```

Reply:
Nice catch ! done